### PR TITLE
package Binaryen

### PR DESCRIPTION
**Pull request by Ludea**
Add Binaryen as a single package. 
More PR are coming to add emscripten

### CODE REVIEW

**Review by buttaface — APPROVED** [review]

File: packages/binaryen/build.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/WebAssembly/binaryen
+TERMUX_PKG_DESCRIPTION="Binaryen is a compiler and toolchain infrastructure library for WebAssembly"
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_VERSION=version_98
```
Comment (Grimler91) [inline packages/binaryen/build.sh:4]: Please change to just 98 (and update the url to be `version_${TERMUX_PKG_VERSION}`)

File: packages/binaryen/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+TERMUX_PKG_SRCURL=https://github.com/WebAssembly/binaryen/archive/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=9f805db6735869ab52cde7c0404879c90cf386888c0f587e944737550171c1c4
+TERMUX_PKG_BUILD_IN_SOURCE=true
```
Comment (Grimler91) [inline packages/binaryen/build.sh:7]: Is this required, have you tried without?

**Review by Grimler91 — CHANGES_REQUESTED** [review]

File: packages/binaryen/build.sh (hunk 2 above)
Comment (Ludea) [reply packages/binaryen/build.sh]: I just try and it's not needed

File: packages/binaryen/build.sh (hunk 1 above)
Comment (Ludea) [reply packages/binaryen/build.sh]: done

**Review by Grimler91 — APPROVED** [review]

**Pull request merged**